devops/aws/templates/Playground-Caddyfile.j2

```diff
@@ -42,3 +42,9 @@
     uri strip_prefix /member-faucet
     reverse_proxy localhost:3002
 }
+
+{{ nip_domain }}/config.json {
+    root * /home/ubuntu
+    rewrite * /config.json
```

Comment:
is it endpoints.json or config.json ?
I wouldn't serve up the whole /home/ubuntu directory, even though I see the rewrite rule :)

Reply:
looks correct actually.